tests/fixtures/data/vocabularies.yaml

```diff
@@ -19,3 +19,6 @@ descriptiontypes:
 relationtypes:
   pid-type: rlt
   data-file: vocabularies/relation_types.yaml
+communitytypes:
```

Comment:
is this needed? IIRC we overwrite in tests when we wanted to load just a subset for tests (e.g. the main languages.yaml has 10k entries but we want just a few for tests). However, in the case of community types is still just 4. Doesn't it pick up defaults if not overwritten here?

Reply:
tried this change locally but tests fail when importing the fixtures.

Reply:
it's because the passed `dir` is the parent (`/tests/fixtures`) and therefore it won't look outside of it. But its not a problem, let's keep it as is.